ansible/playbook/passstore_controller_inventory_remove.yml

```diff
@@ -1,21 +1,24 @@
 ---
-# file: clear_local_inventory_configuration.yml
+# file: passstore_controller_inventory_remove.yml
 # Required variables:
 #  . vm_name: Name of the vm
 #  . pass_provider: provider in the passstore project [hetzner]
 - name: Delete local inventory configuration
   hosts: localhost
   gather_facts: no
-  vars_prompt:
-    - name: vm_name
-      prompt: vm name
-      private: no
-      when: " vm_name is not defined "
-      
-    - name: pass_provider
-      prompt: Provider [hetzner,...]
-      private: no
-      when: "pass_provider is not defined "
+
+  pre_tasks:
+    - name: "Validate required variables"
+      assert:
+        that:
+          - "vm_name is defined"
+          - "pass_provider is defined"
+        fail_msg: "'vm_name' and 'pass_provider' must be defined"
+
+    - name: "Pull pass git database"
+      shell: "git pull"
```

Comment:
How do you plan to return the error to the user's console if `git pull` fails due to merge conflict ?

Reply:
Ansible will stop the process by itself showing the error.

Reply:
LGTM